index.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to the Standard for Public Code.
 
-We define ‘public code’ as open source software developed by public organizations, together with the policy and guidance needed for reuse.
+We define ‘public code’ as open source software developed by public organizations, together with the policy and guidance needed for collaboration.
```

Comment:
just "collaboration" or "collaboration and reuse"?

Reply:
Yes, that might be even better.